### ISSUE TITLE
Notebook-selection not working as expected

### ISSUE THREAD

**Issue by SZenglein** [konradrenner/kolabnotes-android#154]
Hey I think the selection of notebooks is broken a little bit. I'm using the f-droid app on an unofficial LineageOS build.

If I select a notebook only the notes of that notebook are shown as expected. But if I refresh other notes are shown too.
If I select notes of all accounts I have to tap the screen once for the note to show up. Strange.

Sorry if the names are not clear, they're freely translated. This behavior is not really bad, but can be quite confusing when you have selected a notebook and there are other notes displayed.

**Comment by konradrenner** [comment]
Hi,
the first thing I was able to reproduce, yep this is a bug, the selection should change to "All notes of account"(sync of just the selected notebook is not possible at the moment).

But I cannot reproduce the other thing. If I selected"notes of all accounts" the list of all notes is displayed and if I tap on one note, it is opened.

I will have a look at the first problem and test the second on another device/emulator.

Thanks for the feedback!
